ts/src/test/scala/ch/epfl/ts/test/component/SimpleTraderTest.scala

```diff
@@ -0,0 +1,104 @@
+package ch.epfl.ts.test.component
+
+import scala.concurrent.duration.DurationInt
+import scala.language.postfixOps
+import org.scalatest.WordSpecLike
+import akka.actor.ActorSystem
+import akka.actor.Props
+import akka.testkit.TestActorRef
+import akka.testkit.TestKit
+import ch.epfl.ts.component.StartSignal
+import ch.epfl.ts.data.Currency
+import ch.epfl.ts.traders.SimpleFXTrader
+import com.typesafe.config.ConfigFactory
+import ch.epfl.ts.indicators.SMA
+import akka.testkit.EventFilter
+
+class SimpleTraderTest extends TestKit(ActorSystem("testSystem", ConfigFactory.parseString(
```

Comment:
Document that this is testing a single scenario and that tests are inter-dependent.